tests/testthat/test-vdiffr.R

```diff
@@ -90,3 +90,72 @@ test_that("plots are rendered correctly", {
       scale_fill_manual(values = c("grey80", "grey20"))
   )
 })
+
+test_that("method which return text works - snapshot", {
```

Comment:
Before, we were testing internal implementation details of ggplot2 by snapshotting the plot data. 
We are instead interested in how the plot looks, and so a visual snapshot test is better.

Reply:
Sounds good.